data/json/professions.json

```diff
@@ -3417,5 +3417,368 @@
+        "name": {
+            "male": "Skater Boy",
+            "female": "Skater Girl"
+        },
+        "description": "You love to skate! At least now the grown-ups aren't telling you where you can't roll.",
+        "points": 1,
+                "skills": [
+            {
+                "level": 1,
+                "name": "dodge"
+            }
+        ],
+        "traits": [
+            "PROF_SKATER"
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "hoodie",
+                "jeans",
+                "socks",
+                "elbow_pads",
+                "knee_pads",
+                "gloves_fingerless",
+                "helmet_skid",
+                "roller_blades",
+                "sports_drink"
+            ],
+            "male": [
+                "briefs"
+            ],
+            "female": [
+                "boy_shorts"
+            ]
+        }
+    },
+    {
+
+            "type": "profession",
+            "ident": "churchkid",
+            "name": "Student of Faith",
+            "description": "Your parents were deeply faithful, and tried to raise you with the same values. Will you lean on faith to make it through the dark days ahead, or is the Cataclysm proof that there can be no loving supreme being?",
+            "points": 0,
+            "items": {
+                "both": [                    
+                    "dress_shirt",
+                    "socks",
+                    "backpack",
+                    "sneakers",
+                    "jeans",
+                    "jacket_light",
+                    "knit_scarf",
+                    "juice",
+                    "sandwich_pbj",
+                    "holy_symbol"
+                ],
+                "male": [
+                    "briefs"
+                ],
+                "female": [
+                    "panties"
+                ]
+            }
+    },
+    {
+        "type": "profession",
+        "ident": "jdelinquent",
+        "name": "Juvenile Delinquent",
+        "description": "You never cared for grown-ups telling you what to do, and that's how you ended up spending most of your days in the principal's office. Now, not needing grown-ups to tell you what to do is the only reason you're alive. Man, you really should've played hooky today.",
+        "points": 1,
+        "skills": [
+            {
+                "level": 1,
+                "name": "gun"
+            },
+            {
+                "level": 1,
+                "name": "dodge"
+            },
+            {
+                "level": 1,
+                "name": "melee"
+            }
+        ],
+        "items": {
+            "both": [
+                "tshirt",
+                "hoodie",
+                "jeans",
+                "socks",
+                "sneakers",
+                "gloves_fingerless",
+                "matches",
+                "hairpin",
+                "hairpin",
```

Comment:
Obviously you're wearing the hairpins so mean Mr. KA101 doesn't take them away for being lockpicking tools!  ;-)